package.json

```diff
@@ -25,8 +29,8 @@
     "test:coverage": "jest --coverage --colors --maxWorkers=2",
     "hygen": "hygen",
     "component:new": "hygen component with-prompt",
-    "publish": "npx semantic-release --no-ci",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "publish": "npm publish"
```

Comment:
We should remove this line because it's redundant!

Reply:
Yes, that's true! Will remove.

Re: Husky, I agree with using lint-staged with husky, that's how I've used husky in the past. Do you think it's worth doing it on this PR?

Reply:
Hmmmm. Actually now that you mention it, if you're down for it, wanna create a separate PR? It can serve as a shining example of what the transition from `husky` standalone to `husky`+`lint-staged` looks like.

Reply:
Sure! I can def do that :)